static/css/cellml.css

```diff
@@ -115,6 +115,14 @@ div[class^="issue"], div[class*=" issue"] > ol {
     font-family: "Lato","proxima-nova","Helvetica Neue",Arial,sans-serif;
 }
 
+.heading1 {
+    color: #444444;
+    font-family: "Roboto Slab","ff-tisa-web-pro","Georgia",Arial,sans-serif;
```

Comment:
Nitpicking: space between font families. Also, probably use double quotes for all font families.